src/server/services/records.js

```diff
@@ -110,6 +114,7 @@ async function recordsForUpload (upload) {
 }
 
 async function recordsForReportingPeriod (tenantId, periodId) {
+  log('recordsForReportingPeriod()')
```

Comment:
Did you mean to include this log?

Reply:
I did!  We have used this pattern in a few places.  The `log` function is a no-op unless the `VERBOSE` environment variable is set, and logging the name of the function calls can be helpful debugging.  🤷